lib/node_modules/@stdlib/stats/incr/nanvariance/docs/types/test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import incrnanvariance from './index' ;
+
+
+// TESTS //
+
+// The function returns an accumulator function...
+{
+	incrnanvariance(); // $ExpectType accumulator
+	incrnanvariance( 0.0 ); // $ExpectType accumulator
+}
+
+// The compiler throws an error if the function is provided invalid arguments...
+{
+	incrnanvariance( '5' as any ); // $ExpectError
```

Comment:
Remove all of these `as any` casts. The entire point of these lines is to test that the compiler errors when provided various non-any input values.